jerry-core/ecma/operations/ecma-container-object.c

```diff
@@ -370,13 +370,20 @@ ecma_op_container_create (const ecma_value_t *arguments_list_p, /**< arguments l
                 || lit_id == LIT_MAGIC_STRING_SET_UL
                 || lit_id == LIT_MAGIC_STRING_WEAKMAP_UL
                 || lit_id == LIT_MAGIC_STRING_WEAKSET_UL);
+  JERRY_ASSERT (JERRY_CONTEXT (current_new_target) != NULL);
 
   ecma_collection_t *container_p = ecma_op_create_internal_buffer ();
 
-  ecma_object_t *object_p = ecma_create_object (ecma_builtin_get (proto_id),
-                                                sizeof (ecma_extended_object_t),
-                                                ECMA_OBJECT_TYPE_CLASS);
+  ecma_object_t *proto_p = ecma_op_get_prototype_from_constructor (JERRY_CONTEXT (current_new_target), proto_id);
 
+  if (JERRY_UNLIKELY (proto_p == NULL))
+  {
+    return ECMA_VALUE_ERROR;
+  }
```

Comment:
Please add an empty line after the block.

Reply:
This request is still valid.